### PR TITLE
SA-535 Added documentation of ingest events

### PR DESCRIPTION
**Pull request by gwalls**

### CODE REVIEW

File: content/api/events-ingest.apib [hunk 1]
```diff
@@ -23,7 +23,12 @@ Please see [Processing Out of Band Errors](#header-processing-out-of-band-errors
 
 ### Processing Out of Band Errors
 
-**Coming Soon**
+Every uploaded batch of events can be tracked via the `id` returned from the API after a successful POST. This batch ID can be used in conjunction with the Events API to
+view the outcome of SparkPost's pre-processing. 
+
+Note: the ingested email events may still take up to two minutes to be viewable
```
Comment (beardyman) [inline content/api/events-ingest.apib]: I'm not sure that I would commit to 2 minutes here.  That very much feels like a guarantee.  I would say a `few moments` or `a short time` unless that commitment was required from someone else.

File: content/api/events.apib [hunk 2]
```diff
@@ -209,6 +209,61 @@ Returns a list of message events that matched the filtered search.
...
+        + Default: `initial`
+    + per_page (number, optional) - Maximum number of results to return per page. Must be between 1 and 10,000.
+        + Default: `1000`
+    + delimiter (string, optional) ... Specifies the delimiter for query parameter lists.
+        + Default: `,`
+    + events (string, optional) - Comma delimited list of event types to search.
+        + Default: `success,error`
+    + event_ids (string, optional) - Comma delimited list of event IDs to search.
+    + batch_ids (string, optional) - Comma delimited list of batch IDs to search.
+    + event_ids (string, optional) - Comma delimited list of event IDs to search.
```
Comment (beardyman) [inline content/api/events.apib]: `event_ids` twice

File: content/api/index.apib [hunk 3]
```diff
@@ -107,7 +107,7 @@ Features that are limited to some accounts will be marked appropriately througho
 To prevent abuse, our servers enforce request rate limiting, which may trigger responses with HTTP status code 429.
 The following endpoints are rate limited:
 
-- `/api/v1/message-events`
+- `/api/v1/events/*`
```
Comment (beardyman) [inline content/api/index.apib]: Was removing `message-events` on purpose? should we keep both events and message-events?  there are plenty of other endpoints that are rate limited as well

Comment (gwalls) [reply content/api/index.apib]: I thought I'd change the example service to one that wasn't being deprecated 😄

**Pull request merged**